virtcontainers/types/sandbox.go

```diff
@@ -161,6 +163,19 @@ func (v *Volumes) String() string {
 	return strings.Join(volSlice, " ")
 }
 
+// HybridVSock defines a hybrid vsocket to communicate between
+// the host and any process inside the VM.
```

Comment:
I would add a brief explanation stating that this is a virtio-vsock implementation based on AF_VSOCK on the guest side and multiple AF_UNIX sockets on the host side iiuc.

Reply:
yeah, thx